bindings/wasm/manifold.d.ts

```diff
@@ -49,12 +69,15 @@ declare class Mesh {
   originalID?: Uint32Array;
   faceID?: Uint32Array;
   halfedgeTangent?: Float32Array;
+  transform?: Float32Array;
   get numTri(): number;
   get numVert(): number;
-  verts(tri: number): Uint32Array<3>;
-  position(vert: number): Float32Array<3>;
+  get numMatrix(): number;
+  verts(tri: number): SealedUint32Array<3>;
+  position(vert: number): SealedFloat32Array<3>;
```

Comment:
Remind me of the advantage of this over what we had previously?

Reply:
Float32Array doesn't have any type arguments, so you can't specify the size of the array this way; this creates a compile error. If it was a regular array you could have used, for example, a tuple by doing `[x: number, y: number, z: number]` instead. This is a typed array though, so you can't use the tuple syntax. Instead, I extended the typed array to have a fixed length by doing something similar to what was done here: https://stackoverflow.com/a/66628633

Reply:
Very nice, thanks for the explanation.